packages/babel-traverse/src/path/index.js

```diff
@@ -7,6 +7,7 @@ import traverse from "../index";
 import Scope from "../scope";
 import * as t from "@babel/types";
 import { path as pathCache } from "../cache";
+import generator from "@babel/generator";
```

Comment:
This is just a devDependency at the moment, so this is going to throw when installed in production.

Reply:
Also is this going to be a circular dep if generator uses traverse?

Reply:
It doesn't, so I think we're fine for the dependency-cycle case. I'm happy with just adding it as a dependency for now.

Reply:
oh, my bad, ive checked `package.json` too quickly, didnt notice its just a devDep - fix is on the way

Reply:
No problem.